src/decoder/frame_info.hh

```diff
@@ -0,0 +1,65 @@
+#ifndef FRAME_INFO_HH
+#define FRAME_INFO_HH
+
+#include <string>
+#include <fstream>
+
+#include "decoder.hh"
+#include "dependency_tracking.hh"
+#include "serialized_frame.hh"
+
+using namespace std;
+
+class FrameInfo
+{
+private:
+  std::vector<uint8_t> frame_;
+  Optional<RasterHandle> output_raster_;
+
+public:
+  std::string frame_name;
+  Optional<std::string> ivf_filename;
+  size_t offset;
+  size_t length;
+  SourceHash source_hash;
+  TargetHash target_hash;
```

Comment:
This breaks the convention throughout the code of having all class member variables end with _. Can you just change these to getters and make them private?